algorithms/max-flow/problems/cowex/cowex.bats

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bats
+
+@test "Test in a graph without eddies" {
```

Comment:
That is something that I saw in the USACO reference that I looked at.  Essentially they are cycles that artificially boost the max flow by flowing into themselves.  I have not found a graph that does this in practice.  I have asked Dr. Dean about finding an example.